net.brinkervii.grapejuice.yml

```diff
@@ -90,8 +90,8 @@ modules:
     sources:
       - type: file
         dest-filename: wine.tar.xz
-        url: https://github.com/Kron4ek/Wine-Builds/releases/download/8.15/wine-8.15-staging-amd64.tar.xz
-        sha256: 0cc40bb91591f01ac1ed0ac410e7c3112f348ce3951f4a49be84c5501b44e012
+        url: https://github.com/Kron4ek/Wine-Builds/releases/download/8.13/wine-8.13-staging-amd64.tar.xz
```

Comment:
If you're going to downgrade Wine, perhaps it'd also make sense to comment out the `x-data-checker` section below this one, just to prevent the Flathub bot from sending pull requests about it.

Reply:
I'll leave the x-data-checker in (so I don't have to re-add it later), but I'll keep in mind to ignore the flathub bot.